querymonitor/querymonitor.pl

```diff
@@ -654,10 +672,10 @@ sub Initialize_SQLITE_Output{
 CREATE TABLE IF NOT EXISTS kv_store(type text,key text, value text);
 INSERT INTO kv_store VALUES 
        ('GENERAL','Analysis_host','$opt{HOSTNAME}')
-	  ,('GENERAL','import date','$opt{STARTTIME_TZ}')
+	  ,('GENERAL','Analysis date','$opt{STARTTIME_TZ}')
 	  ,('GENERAL','processing file','$opt{ANALYZE}')
 	  ,('GENERAL','Analysis version','$main::VERSION');
-	
```

Comment:
Some weird looking formatting inconsistencies here - caps / no caps, spaces vs. underscores, that kind of thing.

Reply:
Fixed